controllers/flinkcluster_converter.go

```diff
@@ -541,3 +558,67 @@ func isStopDelayExpired(
 	return now.After(
 		lastUpdateTime.Add(time.Duration(delayMinutes) * time.Minute))
 }
+
+func getFlinkConfRsc(clusterName string) (*corev1.Volume, *corev1.VolumeMount) {
+	var confVol *corev1.Volume
+	var confMount *corev1.VolumeMount
+	confVol = &corev1.Volume{
+		Name: flinkConfigMapVolume,
+		VolumeSource: corev1.VolumeSource{
+			ConfigMap: &corev1.ConfigMapVolumeSource{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: getConfigMapName(clusterName),
+				},
+			},
+		},
+	}
+	confMount = &corev1.VolumeMount{
+		Name:      flinkConfigMapVolume,
+		MountPath: flinkConfigMapPath,
+	}
+	return confVol, confMount
+}
+
+// TODO: Wouldn't it be better to create a file, put it in an operator image, and read from them?.
+// Provide logging profiles
```

Comment:
I think it is a good idea to have default config files included in the operator image.